.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -23,16 +23,13 @@ A clear and concise description of what you expected to happen.
 **Screenshots**
 If applicable, add screenshots to help explain your problem.
 
+**Release**
+Please specify the version of Kitodo.Production in which the bug was encountered [e.g. 3.3.1, 3.4.3 etc]
+
 **Desktop (please complete the following information):**
  - OS: [e.g. iOS]
  - Browser [e.g. chrome, safari]
  - Version [e.g. 22]
 
-**Smartphone (please complete the following information):**
```

Comment:
The smartphone section must not be removed but maybe only renamed to "non-desktop (tablet, mobile phone, ..)" or merge both sections to one (f.e. device) and add a field for entering the used device (desktop, tablet, mobile phone, ...) so it is maybe more useful in future if other devices are present.